src/core/CoolBoard.h

```diff
@@ -72,30 +75,34 @@ class CoolBoard {
   bool mqttPublish(String data);
   bool mqttListen();
   void mqttCallback(char *topic, byte *payload, unsigned int length);
+  void otaUpdate();
 
 private:
-  CoolFileSystem fileSystem;
-  CoolBoardSensors coolBoardSensors;
-  CoolBoardLed led;
-  CoolTime rtc;
-  CoolWifi wifiManager;
-  Jetpack jetPack;
-  Irene3000 irene3000;
-  ExternalSensors externalSensors;
-  CoolBoardActuator onBoardActuator;
-  CoolPubSubClient mqttClient;
-  WiFiClient wifiClient;
+  CoolBoardSensors *coolBoardSensors = new CoolBoardSensors;
+  CoolBoardLed *led = new CoolBoardLed;
+  CoolTime *rtc = new CoolTime;
+  CoolWifi *wifiManager = new CoolWifi;
+  Jetpack *jetPack = new Jetpack;
+  Irene3000 *irene3000 = new Irene3000;
+  ExternalSensors *externalSensors = new ExternalSensors;
+  CoolBoardActuator *onBoardActuator = new CoolBoardActuator;
+  CoolPubSubClient *mqttClient = new CoolPubSubClient;
+  WiFiClient *wifiClient = new WiFiClient;
   bool ireneActive = false;
   bool jetpackActive = false;
   bool externalSensorsActive = false;
   bool sleepActive = true;
   bool manual = false;
+  bool doOta = false;
   unsigned long logInterval = 3600;
   unsigned long previousLogTime = 0;
   String mqttId;
   String mqttServer;
   String mqttInTopic;
   String mqttOutTopic;
```

Comment:
no need to initialize everything to `""` by default, `String fwVersion;` is enough